### DISCUSSION TITLE
i need help with bootloader mode

### DISCUSSION THREAD

**Discussion by NommarsX** [shanteacontrols/OpenDeck#207] [General]
Hi, I tried to update opendeck M v2.2.1 from the web editor, I can't get out of safe boot mode, I don't know how to manually reboot the board as removing power doesn't solve anything, please I need help with this, it was my first project and I can't continue, thank you.

**Comment by paradajz** [comment]
What operating system do you use? How exactly did you start the firmware update process?

**Comment by NommarsX** [comment]
Hello, Thank you for your quick response.
I work with mac os big sur, I tried to update the firmware through the web
configuration utility, it updated and I can't get out of bootloader mode.
Thank you!

**Reply by eurorackworld** [reply]
sounds a bit like my issue. I have a mac too, and once had an issue, we found out it could be a mac thing. Are you able to try the same process on Windows?

**Comment by paradajz** [comment]
On the M board there are two 2-pin headers below the USB connector: the left one (BOOT) and the right one (RPB). Short circuit the right one and then connect the board to your PC. A new disk drive will appear. Remove the short circuit. Drag the opendeck22.uf2 file to a newly shown disk drive and wait until the board reboots.

You can get the uf2 file from here: https://github.com/shanteacontrols/OpenDeck/releases/tag/v7.5.1

**Comment by NommarsX** [comment]
Hello, sorry for taking so long to answer, I was on a work trip, solved,
Thanks for the help, I wanted to take the opportunity to ask something
else, maybe a little silly question but I'm very new to this and do not
know very well how I can program the board through sysex, with what app or
tool. Thanks for the help and congratulations for your work!
